src/libraries/Common/src/Interop/Windows/Kernel32/Interop.RaiseFailFastException.cs

```diff
@@ -25,23 +26,33 @@ internal unsafe struct EXCEPTION_RECORD
 
     internal partial class Kernel32
     {
-        internal const uint EXCEPTION_NONCONTINUABLE = 0x1;
-
-        internal const uint FAIL_FAST_GENERATE_EXCEPTION_ADDRESS = 0x1;
+        private const uint EXCEPTION_NONCONTINUABLE = 0x1;
+        private const uint FAIL_FAST_GENERATE_EXCEPTION_ADDRESS = 0x1;
+        private const uint STATUS_STACK_BUFFER_OVERRUN = 0xC0000409;
+        private const uint FAST_FAIL_EXCEPTION_DOTNET_AOT = 72;
```

Comment:
Nit: It is common to write these subcodes as decimal or as hex?

I see that windbg prints them as hex (`Subcode: 0x48 (unknown subcode)`).

Reply:
It seems that Andrew uses 72 directly in the provider/!analyze code:

https://microsoft.visualstudio.com/OS/_git/os.2020?path=/sdktools/debuggers/exts/badev/CLRMA/clrmaAOT/AOTHelper.cpp&version=GCafe78c8c12acbccffeecdd398cb1f99a7ab3227b&line=81&lineEnd=82&lineStartColumn=1&lineEndColumn=1&lineStyle=plain&_a=contents

https://microsoft.visualstudio.com/OS/_git/os.2020?path=/sdktools/debuggers/exts/badev/CLRMA/clrmaAOT/AOTHelper.cpp&version=GCafe78c8c12acbccffeecdd398cb1f99a7ab3227b&line=81&lineEnd=82&lineStartColumn=1&lineEndColumn=1&lineStyle=plain&_a=contents

But I'm fine with coding the const as hex.

Reply:
Either way is fine with me. I was just wondering.